### ISSUE TITLE
client version 1.11 is too old. Minimum supported API version is 1.12

### ISSUE THREAD

**Issue by edog305** [Transmode/gradle-docker#75]
Hi,

I'm getting the following error trying to connect to a remote docker-machine:

Docker API error: Failed to build Image:
client version 1.11 is too old. Minimum supported API version is 1.12, please upgrade your client to a newer version

Does this plugin not support version higher than 1.11?

Thanks!

**Comment by edog305** [comment]
I just found that the cause is probably due to the docker-java dependency being very old.

**Comment by mattgruter** [comment]
We just upgraded to docker-java dependency to version 2.2.0 on master. The change will be in version 1.3.

**Comment by dmakshay** [comment]
@mattgruter when can we expect v1.3 release? Waiting eagerly since even I'm getting the same error.

**Comment by SreekarVanguru** [comment]
@mattgruter Just checking when can we expect v1.3 ?

**Comment by weltraumpirat** [comment]
@mattgruter This is also a huge show stopper for me - can I help with any open issues that need to go into the 1.3 release?

**Comment by Datenheld** [comment]
Can you please fix this?

**Comment by blub0hr** [comment]
Any chance to get 1.3 released after over a year now?

**Comment by bjornmagnusson** [comment]
Yes very soon. Some final issue to look over and verification to be done

**Comment by stanislasbonifetto** [comment]
I have the same issue.
Can you release a hotfix?
